### PR TITLE
Fix attribute code formatting and a typo in yaf

### PR DESCRIPTION
**Pull request by kamil-tekiela**
Also, fixes typo mentioned in the chat https://chat.stackoverflow.com/transcript/message/51088431

### CODE REVIEW

File: language/attributes.xml [hunk 1]
```diff
@@ -44,53 +44,50 @@
...
+    #[SetUp]
+    public function fileExists() {
+        if (!file_exists($this->fileName)) {
+            throw new RuntimeException("File does not exist");
+        }
+    }
+
+    #[SetUp]
+    public function targetDirectoryExists() {
+        mkdir($this->targetDirectory);
```
Comment (cmb69) [inline language/attributes.xml:69]: Not sure if the `@` is used deliberately; maybe an already existing targetDirectory is fine?

**Review by cmb69 — COMMENTED** [review]
Thank you! I have two concerns, but otherwise this looks fine.

File: language/attributes.xml (hunk 1 above)
Comment (kamil-tekiela) [reply language/attributes.xml:69]: Then it should be wrapped in `if (!file_exists($this->targetDirectory))`. An error suppression operator should almost always be avoided. If used, then a comment should be present explaining why this particular line can be implicitly trusted to be always right.

File: language/attributes.xml [hunk 2]
```diff
@@ -44,53 +44,50 @@
      <programlisting role="php">
 <![CDATA[
 <?php
-interface ActionHandler
-{
+
+interface ActionHandler {
```
[the remark this answers is not in the document]
Comment (kamil-tekiela) [reply language/attributes.xml]: Fixed. I changed the style to PEAR CS. FWIW the page was not consistent before and I was not sure which one is more appropriate so I went for the less verbose style.

File: language/attributes.xml (hunk 1 above)
Comment (cmb69) [reply language/attributes.xml]: Fair enough.

File: language/attributes.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (cmb69) [reply language/attributes.xml]: Ah, indeed, was inconsistent before; thanks for fixing!

**Pull request closed without merging**